runtime/classlib.properties

```diff
@@ -1,7 +1,29 @@
-# Properties file for Java 9 and beyond
+################################################################################
+# Copyright (c) 2019, 2019 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+################################################################################
+
+# Properties file for Java 8 (Java 11 and beyond doesn't need this file)
 
 # Shape of the class library (e.g. Sun.Basic, Harmony)
 shape=vm.shape
 
 # Version of the class library
-version=1.9
+version=1.8
```

Comment:
What is this file used for? It doesn't define the bootclasspath.

Reply:
Please see https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/273.
`Java 8` extension hardcoded `-e 's/shape=vm.shape/shape=sun_openjdk\nbootpath=rt.jar:resources.jar:jsse.jar:charsets.jar:jce.jar/g'`.

Reply:
Just seems weird that the only purpose of the file in OpenJ9 is so the extensions can modify it.

Reply:
Probably it was for customization accordingly to Java version, now `Java 8` is the only user. `
bootpath` can be hardcoded here within `OpenJ9` and the `Java 8` extension script can just copy it without modification.

Reply:
Technically the extensions owns the bootclasspath, as the extensions define and create the jar files. If it were to be changed, I'd remove the file from OpenJ9 and create it entirely from the extensions.

Reply:
In that case, I would like leave such change to another PR and allow this PR merge to unblock `JDK13` build.